assets/blocks/quiz/quiz-block/quiz-settings.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * WordPress dependencies
+ */
+import { InspectorControls } from '@wordpress/block-editor';
+import {
+	PanelBody,
+	PanelRow,
+	RangeControl,
+	ToggleControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import NumberControl from '../../editor-components/number-control';
+
+/**
+ * Quiz settings.
+ *
+ * @param {Object}   props                    Block props.
+ * @param {Object}   props.attributes         Block attributes
+ * @param {Object}   props.attributes.options Current setting options.
+ * @param {Function} props.setAttributes      Set attributes function.
+ */
+const QuizSettings = ( { attributes: { options = {} }, setAttributes } ) => {
+	const {
+		passRequired = false,
+		quizPassmark = 100,
+		autoGrade = true,
+		allowRetakes = true,
+		randomQuestionOrder = false,
+		showQuestions = null,
+	} = options;
+
+	const createChangeHandler = ( optionKey ) => ( value ) =>
+		setAttributes( { options: { ...options, [ optionKey ]: value } } );
+
+	return (
+		<InspectorControls>
+			<PanelBody
+				title={ __( 'Quiz Settings', 'sensei-lms' ) }
+				initialOpen={ true }
+			>
+				<PanelRow>
+					<ToggleControl
+						checked={ passRequired }
+						onChange={ createChangeHandler( 'passRequired' ) }
+						label={ __( 'Pass Required', 'sensei-lms' ) }
+					/>
+				</PanelRow>
+				{ passRequired && (
+					<PanelRow>
+						<RangeControl
+							label={ 'Passing Grade' }
+							value={ quizPassmark }
+							onChange={ createChangeHandler( 'quizPassmark' ) }
+							min={ 0 }
+							max={ 100 }
+							initialPosition={ 100 }
+						/>
+					</PanelRow>
+				) }
+				<PanelRow>
+					<ToggleControl
+						checked={ autoGrade }
+						onChange={ createChangeHandler( 'autoGrade' ) }
+						label={ __( 'Auto Grade', 'sensei-lms' ) }
+						help={ __(
+							'Only applicable for multiple choice, true/false abd gap fill questions.',
```

Comment:
Typo here ~~`abd`~~ > `and`. But maybe we can keep the original explanation here, as it has a lot of extra information:

> Grades quiz and displays answer explanation immediately after completion. Only applicable if quiz is limited to Multiple Choice, True/False and Gap Fill questions. Questions that have a grade of zero are skipped during autograding.

Reply:
Help text updated here: https://github.com/Automattic/sensei/pull/3987/commits/136e5640d2675df20b6413c4864158504364913f